modules/ipi-install-configuring-bios-for-worker-node.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-bios-for-worker-node_{context}"]
+= Configuring BIOS for worker node
+
+During the installation process, the BIOS of the worker node can be configured
+by the following steps.
+
```

Comment:
Are there any prerequisites for this procedure that needs to be included? Such as needing `cluster-admin` privileges?

Reply:
There is no prerequisite, which is the normal cluster deployment through IPI.